app/src/components/LayerControlBody.vue

```diff
@@ -610,7 +710,43 @@ export default {
         name: `flip on axis ${axis}`
       })
       this.flipAxis({ axis })
+    }, 
+    setResolution: function () {
```

Comment:
[minor point] it is slightly awkward to use the oninput to trigger state update. 

It would probably more declarative to use a `watch: { selectedResolutionX }`, or indeed, if we are using computed properties, a setter method for the computed property to trigger state update.

semantically, it makes more sense (e.g. anything in the component can trigger a method call. But if the update state is hooked to a watch hook or setter, then *only* value update triggers the state update)